build-deploy-platform-services/Jenkinsfile

```diff
@@ -51,7 +55,7 @@ def Event_Status = [
 node {
 
 	echo "params : $params"
-	def jazzBuildModuleURL = "http://${env.REPO_BASE}/scm/${env.REPO_CORE}/jazz-build-module.git"
+	def jazzBuildModuleURL = "http://" + repo_base + "/scm/" + repo_core + "/jazz-build-module.git"
```

Comment:
use same convention of single string with variables => "http://${repo_base}/scm/${repo_core}/jazz-build-module.git"

Reply:
completed